lib/ood_core/job/adapters/kubernetes/templates/pod.yml.erb

```diff
@@ -89,9 +94,15 @@ spec:
     <%- end # command loop -%>
     volumeMounts:
     <%- unless configmap.nil? -%>
+    <%- configmap.files.each do |file| -%>
+    <%- next if file.init_mount_path == false -%>
     - name: configmap-volume
-      mountPath:  <%= configmap_mount_path %>
-    <%- end -%>
+      mountPath: <%= file.init_mount_path || configmap_mount_path %>
```

Comment:
Yes, because if you define nothing it's `nil` and that is going to be when it defaults to `configmap_mount_path` which is what the unit tests are testing. All test YAML files have the configmap mounted at `/ood` because `init_mount_path` is `nil`, except where it's set to `false` explicitly in one of the tests.

Reply:
Oh I see that test now. I'll create a follow up ticket because it's not super intuitive where everything is going. I used `/ood` to begin with, but maybe we need to get rid of that and actually specify the path for configmaps.